sass/main.scss

```diff
@@ -58,10 +58,21 @@
 
 
 .filter-group {
+  margin-top: 30px;
```

Comment:
This is in "our own" already. As for the file type, I'm okay with either "sass" or "scss", I'll change it to sass.